README.md

```diff
@@ -29,6 +29,68 @@ bundle install
 
 Note that `bundle install` may complain if MySQL isn't installed.  You can either comment out the `mysql2` inclusion in `Gemfile` and come back to it later (you can develop using `sqlite3`), or you can install MySQL.
 
+
+## Run the tests locally
```

Comment:
Not a blocker, but you may want `docker-compose down` as this nukes the volumes for solr & fedora and allows you to start afresh.